docs/PowerAuth-Server-1.6.0.md

```diff
@@ -4,6 +4,23 @@ This guide contains instructions for migration from PowerAuth Server version `1.
 
 ## Database Changes
 
+### Allow Non-personalized Operations
```

Comment:
I have noticed that this file is not linked from `Migration-Instructions.md` yet.